Edit review of Execution section (#13253)
PrefectHQ/prefect: docs/guides/dask-ray-task-runners.md

    To run tasks asynchronously use the `.submit` method when you call them. If you call a task as you would normally in Python code it will run synchronously, even if you are calling the task within a flow that uses the `ConcurrentTaskRunner`, `DaskTaskRunner`, or `RayTaskRunner`.

Many real-world data workflows benefit from truly parallel, distributed task execution. For these use cases, the following Prefect-developed task runners for parallel task execution may be installed as [Prefect Integrations](/integrations/catalog/).

- [`DaskTaskRunner`](https://prefecthq.github.io/prefect-dask/) runs tasks requiring parallel execution using [`dask.distributed`](http://distributed.dask.org/).
- [`RayTaskRunner`](https://prefecthq.github.io/prefect-ray/) runs tasks requiring parallel execution using [Ray](https://www.ray.io/).

These task runners can spin up a local Dask cluster or Ray instance on the fly, or let you connect with a Dask or Ray environment you've set up separately. This enables you to take advantage of massively parallel computing environments.

Use Dask or Ray in your flows to choose the execution environment that fits your particular needs.

!!! note "Remote storage"

>     We recommend configuring [remote file storage](/concepts/storage/) for task execution with `DaskTaskRunner` or `RayTaskRunner`. This ensures tasks executing in Dask or Ray have access to task result storage, particularly when accessing a Dask or Ray instance outside of your execution environment.

Reviewer: ```suggestion
    To ensure that tasks executing in Dask or Ray have access to task result storage, particularly when accessing a Dask or Ray instance outside of your execution environment, configure [remote file storage](/concepts/storage/).
```

## Configure a task runner

> Configure a specific task runner for a flow.

Reviewer 2: ```suggestion
Configure a task runner for a flow.
```

The tasks are not submitted to the `DaskTaskRunner` and are run sequentially.

## Run parallel tasks with Ray

To flexibly apply the appropriate task runner for your workflow, use the same flow as above, with a few minor changes to use the [`RayTaskRunner`](https://prefecthq.github.io/prefect-ray/) where you previously configured `DaskTaskRunner`. 

Configure your flow to use the `RayTaskRunner`:

1. Make sure the `prefect-ray` collection is installed by running `pip install prefect-ray`.
2. In your flow code, import `RayTaskRunner` from `prefect_ray.task_runners`.
3. Assign it as the task runner when the flow is defined using the `task_runner=RayTaskRunner` argument.

!!! warning "Ray environment limitations"
    While we're excited about parallel task execution through Ray to Prefect, there are some inherent limitations with Ray:
    

>     - Support for Python 3.11 is [experimental](https://docs.ray.io/en/latest/ray-overview/installation.html#install-nightlies).

Reviewer 2: - Ray currently officially supports x86_64, aarch64 (ARM) for Linux, and Apple silicon (M1) hardware.

>     - Ray support for non-x86/64 architectures such as ARM/M1 processors with installation from `pip` alone and will be skipped during installation of Prefect. It is possible to manually install the blocking component with `conda`. See the [Ray documentation](https://docs.ray.io/en/latest/ray-overview/installation.html#m1-mac-apple-silicon-support) for instructions.

Reviewer 2: ```suggestion
```

    greetings(["arthur", "trillian", "ford", "marvin"])
```

Now run `ray_flow.py` `RayTaskRunner`, which automatically creates a local Ray instance, then immediately starts executing all of the tasks in parallel. If you have an existing Ray instance, you can provide the address as a parameter to run tasks in the instance. See [Running tasks on Ray](/concepts/task-runners/#running_tasks_on_ray) for details.

## Use multiple task runners

Many workflows include a variety of tasks, and not all of them benefit from parallel execution. You'll most likely want to use the Dask or Ray task runners and spin up their respective resources only for those tasks that need them.

Because task runners are specified on flows, you can assign different task runners to tasks by using [subflows](/concepts/flows/#composing-flows) to organize those tasks.

> This example uses the same tasks as the previous examples, but on the parent flow `greetings()` you use the default `ConcurrentTaskRunner`. Then you call a `ray_greetings()` subflow that uses the `RayTaskRunner` to execute the same tasks in a Ray instance.

Reviewer 2: ```suggestion
This example uses the same tasks as the previous examples, but on the parent flow, `greetings()`, you use the default `ConcurrentTaskRunner`. Then you call a `ray_greetings()` subflow that uses the `RayTaskRunner` to execute the same tasks in a Ray instance.
```
